### DISCUSSION TITLE
Как сделать подсказку при клике по иконке i (Информация)?

### DISCUSSION THREAD

**Discussion by Dazdranaganj** [twbs/bootstrap#37188] [Q&A]
Всем доброго времени суток. Подскажите пожалуйста как сделать всплывающую подсказку при клике по иконке i (информация).  Везде приводят примеры где подсказка всплывает при наведении на ссылку, а мне нужно при клике на неё.
![i](https://user-images.githubusercontent.com/44741314/191626214-1b7c3ce6-67ef-4f3a-81a4-6a521f93db85.png)
Штатный момент для любого сайта - дополнительная информация.  А как это сделать в Bootstrap?

**Comment by coliff** [comment]
Hi - I think you can create something like that using custom popovers (which activate on click be default).

See https://getbootstrap.com/docs/5.2/components/popovers/#custom-popovers for examples.
